ddtrace/span.py

```diff
@@ -310,7 +312,7 @@ def set_tag(self, key, value=None):
 
     def _set_str_tag(self, key, value):
```

Comment:
Can we add to the docstring that errors in decoding will be replaced with ? as per https://docs.python.org/3/library/codecs.html#codecs.replace_errors